src/main/java/de/tum/in/www1/artemis/domain/StudentQuestionAnswer.java

```diff
@@ -6,6 +6,7 @@
 
 import org.hibernate.annotations.Cache;
 import org.hibernate.annotations.CacheConcurrencyStrategy;
+import org.springframework.cloud.cloudfoundry.com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
I don't think this is the correct import

Reply:
resolved with d6b9cc6